.gitignore

```diff
@@ -75,6 +75,7 @@ data/
 
 # custom Sentry config
 sentry/sentry.conf.py
+sentry/*.bak
```

Comment:
Nit: this should probably come after `sentry/config.yml` to make the comment above relevant.

Reply:
Still stands as a nit.

Reply:
Yes, that was sloppy of me